coilsnake/ui/gui.py

```diff
@@ -350,6 +378,12 @@ def create_gui(self):
         decompile_script_frame = self.create_decompile_script_frame(self.notebook)
         self.notebook.add(decompile_script_frame, text="Decompile Script")
 
+        patcher_patch_frame = self.create_patcher_patch_frame(self.notebook)
+        self.notebook.add(patcher_patch_frame, text="Patch ROM")
```

Comment:
I'd suggest calling this "Apply Patch" so that it's more clear what this does versus the "Create Patch" option in the future